knowledge-base/dropdownbutton-add-separator-between-items.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Adding a Separator Between Items in a DropDownButton for Blazor
+description: Learn how to add a separator between the items in a DropDownButton by utilizing custom styling in a Blazor application.
+type: how-to
+page_title: How to Implement Item Separators in a Telerik Blazor DropDownButton
+slug: dropdownbutton-kb-add-separator-between-items
+tags: dropdownbutton, blazor, item separator, custom styling
+res_type: kb
+ticketid: 1680307
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>DropDownButton for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+## Description
+
+I want to visually separate the items in a [DropDownButton](slug:dropdownbutton-overview) to enhance the UI and improve user experience. How to add a separator (divider) between the items?
+
+## Solution
+
+The DropDownButton does not have a concept for a "separator" item (similar to the [ContextMenu](slug:contextmenu-data-binding-overview), for example). However, the DropDownButton uses a declarative approach for its items, so you can add any desired item.
+
+The specific is that the content you declare in the DropDownButtonItem will be rendered inside the `<span class="k-menu-link-text">` which is narrower and thus your custom separator will not cover the whole width of the popup. This article demonstrates how to add a separator by utilizing custom styling.
```

Comment:
This paragraph confused me at first reading. The first and second sentence must be reworded, so that it's clear that the second sentence presents an alternative to the problem in the first one.

"Use" is simpler than "utilize".

```suggestion
The content of the DropDownButtonItem is rendered inside a `<span class="k-menu-link-text">` element, which is narrower. Thus, a border inside the dropdown item content cannot span over the whole width of the popup. This article demonstrates how to add a separator by setting a CSS class on the whole item and using custom styling.
```